view_user_cert.php

```diff
@@ -0,0 +1,110 @@
+<?php
+// This file is part of Moodle - https://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <https://www.gnu.org/licenses/>.
+// Include required Moodle configuration and custom certificate library.
+
+require_once(__DIR__ . '/../../config.php');
+require_once($CFG->dirroot . '/mod/customcert/lib.php');
+
+// Allows guest access to course with ID 1.
+require_login(1, false);
```

Comment:
Does this line cause this script to only work if the site is configured with "allow guest login" ?

Are there any other situations where this line of code would cause the certificate to not be displayed?